### ISSUE TITLE
Prod build is failing with Directive SelectableRow, Expected 0 arguments, but got 1.

### ISSUE THREAD

**Issue by kewalpasad** [primefaces/primeng#10180]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search GitHub for a similar issue or PR before submitting
[ ] feature request => Please check if the request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit a support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```
**Current behavior**
I have added the selectableRow directive to my p-table. It is behaving as expected but when I build it with --prod it is throwing `Directive SelectableRow, Expected 0 arguments, but got 1.` so I did some digging looks like the problem is something to do with the AOT compiler as pointed out here `https://stackoverflow.com/questions/66161266/library-production-build-failed-on-angular-11-with-expected-0-arguments-but-go`.

**Expected behavior**
should not through any error while building with --prod flag

* **Angular version:** 5.X*
Angular CLI: 9.1.5
Node: 10.19.0 

* **PrimeNG version:** 5.X
PrimeNG version11.3.1

* **Browser:** 
Chrome
 
* **Language:** 
TypeScript 3.8.3

* **Node (for AoT issues):** `node --version` =  10.19.0

**Comment by ruchiagrawal2** [comment]
I am getting the same error in angular 10, prime ng 11.4.1 . How can I resolve this error by still keeping enableIvy=false. Thank You.

**Comment by kewalpasad** [comment]
it should be fixed now the fix was added to the repo by @yigitfindikli

**Comment by ruchiagrawal2** [comment]
@kewalpasad Thanks. @yigitfindikli  Do I need to wait for 11.4.1 to get this fix ? When is the expected release of 11.4.1 ? any work around ?

**Comment by yigitfindikli** [comment]
@kewalpasad thanks for the report!
@ruchiagrawal2 V11.4.1 will be released next week.
